tests/load/inputs/preauth_cart.js

```diff
@@ -0,0 +1,24 @@
+import { create_amount } from "./amount.js";
+import { address } from "./address.js";
+import { create_test_item } from "./test_item.js";
+import { shipment } from "./shipment.js";
+
+export function create_preauth_cart ( order_reference ) {
+    return {
+        "order_reference": order_reference,
+        "display_id": "100044865|" + order_reference,
```

Comment:
In reality, the number after the pipe will not be the `order_reference` but rather a similar number that is greater than the `order_reference`, typically by 1 or 2.  I'm not sure this affects the test however.

Reply:
Interesting, but yeah I double checked and it shouldn't affect the test